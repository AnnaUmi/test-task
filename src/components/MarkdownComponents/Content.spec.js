import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Content from './Content.styles';

describe('Content', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Content />);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
