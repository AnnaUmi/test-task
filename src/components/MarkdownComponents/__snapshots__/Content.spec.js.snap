// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should match snapshot 1`] = `
.c0 {
  padding: 0 20px;
  width: 100%;
}

<div
  className="c0"
/>
`;
